tuning/data/data_config.py

```diff
@@ -153,6 +154,18 @@ def _validate_dataprocessor_config(dataprocessor_config) -> DataPreProcessorConf
         chat_template = kwargs["chat_template"]
         assert isinstance(chat_template, str), "chat_template should be a string"
         c.chat_template = chat_template
+    elif "chat_template_base64" in kwargs:
+        chat_template_base64 = kwargs["chat_template_base64"]
+        assert isinstance(
+            chat_template_base64, str
+        ), "chat_template_base64 should be a string"
+        logger.warning(
+            "You are using the 'chat_template_base64' field. "
+            + "Please use the 'chat_template' field instead for better readability."
+        )
+        chat_template_bytes = b64decode(chat_template_base64)
+        chat_template = chat_template_bytes.decode("utf-8")
```

Comment:
Always `utf-8`?

Reply:
yes, the `chat_template` is always a valid utf-8 string.